dora/src/bytecode/generator_tests.rs

```diff
@@ -1190,11 +1205,17 @@ fn gen_throw() {
                 Throw(r(0)),
                 RetVoid,
             ];
+            let expected_position = vec![(0, p(2, 38)), (6, p(2, 38)), (10, p(2, 23))];
```

Comment:
The position is depending on the position inside the rust source. Shouldn't be a problem, unless the code is formatted...